src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6009,7 +6059,7 @@ namespace SimAirServingZones {
 
 			// write predefined standard 62.1 report data
 			for ( AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
-				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {
+//				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {  // commented line allows ZoneSum method to report tables to the html file
```

Comment:
Unit test OutputReportTabular_ZoneMultiplierTest shows the ACH in the OA Summary Report are the same for non-multiplied and multiplied zones.

![oasummary](https://cloud.githubusercontent.com/assets/5778156/11201048/a7b5f3f6-8ca8-11e5-8df1-dae881a99907.png)